tests/unit/test_github_enterprise.py

```diff
@@ -0,0 +1,134 @@
+from github3.github import GitHubEnterprise
+import github3
+
+from .helper import UnitHelper
+
+example_data = {
+    "login": "octocat",
+    "id": 1,
+    "avatar_url": "https://github.com/images/error/octocat_happy.gif",
+    "gravatar_id": "somehexcode",
+    "url": "https://api.github.com/users/octocat",
+    "html_url": "https://github.com/octocat",
+    "followers_url": "https://api.github.com/users/octocat/followers",
+    "following_url": ("https://api.github.com/users/octocat/following"
+                      "{/other_user}"),
+    "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
+    "starred_url": ("https://api.github.com/users/octocat/starred"
+                    "{/owner}{/repo}"),
+    "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
+    "organizations_url": "https://api.github.com/users/octocat/orgs",
+    "repos_url": "https://api.github.com/users/octocat/repos",
+    "events_url": "https://api.github.com/users/octocat/events{/privacy}",
+    "received_events_url": ("https://api.github.com/users/octocat/"
+                            "received_events"),
+    "type": "User",
+    "site_admin": False,
+    "name": "monalisa octocat",
+    "company": "GitHub",
+    "blog": "https://github.com/blog",
+    "location": "San Francisco",
+    "email": "octocat@github.com",
+    "hireable": False,
+    "bio": "There once was...",
+    "public_repos": 2,
+    "public_gists": 1,
+    "followers": 20,
+    "following": 0,
+    "created_at": "2008-01-14T04:33:35Z",
+    "updated_at": "2008-01-14T04:33:35Z"
+}
+
+base_url = 'https://ghe.example.com/'
+
+
+def _build_url(_, *args, **kwargs):
+    return github3.session.GitHubSession().build_url(
+        base_url=base_url + 'api/v3', *args, **kwargs)
+
+
+class TestGitHubEnterprise(UnitHelper):
+    described_class = GitHubEnterprise
+    example_data = base_url
+
+    def create_instance_of_described_class(self):
+        instance = self.described_class(self.example_data)
+        instance.session = self.session
+
+        return instance
+
+    def after_setup(self):
+        self.described_class._build_url = _build_url
+
+    def url_for(self, path=''):
+        """Generate URLs with the base GitHubEnterprise URL."""
+        return base_url + 'api/v3/' + path.strip('/')
+
+    def test_create_user(self):
+        """Show that an admin can ask for user creation."""
+        self.instance.create_user('login_test', 'email_test')
+
+        self.post_called_with(
+            self.url_for('admin/users'),
+            data={'login': 'login_test', 'email': 'email_test'}
+        )
+
+
+class TestUserAdministration(UnitHelper):
+    described_class = github3.users.User
+    example_data = example_data.copy()
```

Comment:
i do not think you need to make a copy of example data...you are not changing it anywhere